topic_message_query.go

```diff
@@ -158,6 +158,8 @@ func (query *TopicMessageQuery) Subscribe(client *Client, onNext func(TopicMessa
 				} else {
 					panic(err)
 				}
+
+				handle.Unsubscribe()
```

Comment:
this line is unreachable if:

1. unrecoverable error, thus on line 153, it breaks out the for loop
2. completion of the subscription, thus on line 157, it break out the for loop